web/src/styles/theme.css

```
@@ -380,4 +380,83 @@
     animation: orb-drift 6s ease-in-out infinite;
     pointer-events: none;
   }
+
+  .md-preview {
+    color: var(--foreground);
+    font-size: 0.8125rem;
+    line-height: 1.6;
+    word-wrap: break-word;
```

Stylelint (17.9.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)